_posts/2018-12-04-imposing-invariants-in-scala.md

```diff
@@ -0,0 +1,359 @@
+---
+layout: post
+author: cmteixeira
+title: Imposing invariants in Scala
+---
+
+
+
+Imagine a opera house allows reservations to a show. For each reservation two people can come: The buyer of the reservation himself, and his guest.  
```

Comment:
Please replace the "T" in `The buyer` with a lower case "t", as that is not the beginning of a new sentence.
Also here `of the reservation himself, and his guest.` you can remove the comma.

Reply:
done.